[apm] Add Resource Panel information to the Service Page docs (#14690)
DataDog/documentation: content/en/tracing/services/service_page.md

{{< img src="tracing/visualization/service/resources_tab.jpg" alt="Resources"  style="width:100%;">}}

> Click on a resource to view a summary panel with the resource’s out-of-the-box graphs (Requests, Errors, Latency), the Resource Dependency Map, and Span Summary Table. Use keyboard navigation keys to toggle between resources on the resource list to compare resources in a service, and quickly identify issues without context switching. Click “Open Full Page” or use cmd-click or cntrl-click to view the full resource page in a new tab.

Reviewer: ```suggestion
Click on a resource to open a side panel that displays the resource's out-of-the-box graphs (requests by version, errors by version, and latency), a resource dependency map, and a span summary table. Use keyboard navigation keys to toggle between resources on the **Resources** list and compare resources in a service. To view the full resource page, click **Open Full Page**.
```

Reviewer 2: "(requests by version, errors by version, and latency)" - other graphs can be shown through the toggle, can we keep it broad to just "(requests, errors, and latency)"?
